homeassistant/helpers/selector.py

```diff
@@ -52,6 +54,8 @@ class EntitySelector(Selector):
             vol.Optional("domain"): str,
             # Device class of the entity
             vol.Optional("device_class"): str,
+            # Allow multiple entity selection
+            vol.Optional("multiselect"): boolean,
```

Comment:
`multiselect` is a bit of a technical term maybe, how about just `multiple`?

Reply:
Changed as suggested